streaming/text/c4.py

```diff
@@ -22,24 +22,24 @@ class C4(Dataset):
         max_seq_len (int): The max sequence length of each token sample.
         group_method (str): How to group text samples into token samples. Currently only supporting
             ``'truncate'``.
-        local (str): Local dataset directory where shards are cached by split.
-        remote (str, optional): Download shards from this remote path or directory. If None, this
-            rank and worker's partition of the dataset must all exist locally. Defaults to
-            ``None``.
-        split (str, optional): Which dataset split to use, if any. Defaults to ``None``.
-        shuffle (bool): Whether to shuffle the samples while iterating. Defaults to ``True``.
-        prefetch (int, optional): Target number of samples remaining to prefetch while iterating.
+        local (str): Local filesystem directory where dataset is cached during operation.
+        remote (str, optional): Remote directory (S3 or local filesystem) where dataset is stored.
             Defaults to ``None``.
-        keep_zip (bool, optional): Whether to keep or delete the compressed file when
-            decompressing downloaded shards. If set to None, keep iff remote is local. Defaults to
+        split (str, optional): The dataset split to use, either 'train' or 'val'. Defaults to
             ``None``.
+        shuffle (bool): Whether to shuffle the samples in this dataset. Defaults to
+            ``True``.
+        prefetch (int, optional): Target number of samples remaining to prefetch while iterating.
+            Defaults to ``100_000``.
+        keep_zip (bool, optional): Whether to keep or delete the compressed file when decompressing
+            downloaded shards. If set to None, keep iff remote is local. Defaults to ``None``.
         retry (int): Number of download re-attempts before giving up. Defaults to ``2``.
-        timeout (float): Number of seconds to wait for a shard to download before raising an
-            exception. Defaults to ``60``.
-        hash (str, optional): Optional hash or checksum algorithm to use to validate shards.
+        timeout (float): Number of seconds to wait for a shard to download before raising
+            an exception. Defaults to ``60``.
+        hash (str, optional): Hash or checksum algorithm to use to validate shards. Defaults to
+            ``None``.
+        batch_size (int, optional): Batch size that will be used on each device's DataLoader.
```

Comment:
may want to leave in the "hinting" part in, as the field is optional

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/42/commits/c023c30b7f55b89dd9f6526480c0c5a272940bba.